string node_name
string param_name
string value
